fixtures/vcr_cassettes/ForgeMatchBitbucket.yml

```diff
@@ -71,4 +71,77 @@ http_interactions:
         true, "no_public_forks": false, "creator": null, "resource_uri": "/1.0/repositories/durin42/hgsubversion"}'
     http_version: 
   recorded_at: Fri, 20 Mar 2015 14:02:30 GMT
+- request:
```

Comment:
Since you have new casettes, you may delete the older one above this. Run the specs and see if it works after removal.